js/pro/hitbtc3.js

```diff
@@ -0,0 +1,1190 @@
+            'info': ticker,
+        }, market);
+    }
+
+    handleTicker (client, message) {
+        //
+        //     {
+        //          "ch": "ticker/1s",
+        //          "data": {
+        //              "ETHBTC": {
+        //                  "t": 1614815872000,             // Timestamp in milliseconds
+        //                  "a": "0.031175",                // Best ask
+        //                  "A": "0.03329",                 // Best ask quantity
+        //                  "b": "0.031148",                // Best bid
+        //                  "B": "0.10565",                 // Best bid quantity
+        //                  "c": "0.031210",                // Last price
+        //                  "o": "0.030781",                // Open price
+        //                  "h": "0.031788",                // High price
+        //                  "l": "0.030733",                // Low price
+        //                  "v": "62.587",                  // Base asset volume
+        //                  "q": "1.951420577",             // Quote asset volume
+        //                  "p": "0.000429",                // Price change
+        //                  "P": "1.39",                    // Price change percent
+        //                  "L": 1182694927                 // Last trade identifier
+        //              }
+        //          }
+        //      }
+        //
+        const channel = this.safeValue (message, 'ch');
+        const tickers = this.safeValue (message, 'data', []);
+        const keys = Object.keys (tickers);
+        for (let i = 0; i < keys.length; i++) {
+            const marketId = keys[i];
+            const ticker = tickers[marketId];
+            const market = this.safeMarket (marketId);
+            const symbol = market['symbol'];
+            const result = this.wsParseTicker (ticker, market);
+            this.tickers[symbol] = result;
+            const messageHash = channel + ':' + marketId;
+            client.resolve (result, messageHash);
+        }
+        return message;
+    }
+
+    async watchOHLCV (symbol, timeframe = '1m', since = undefined, limit = undefined, params = {}) {
+        /**
+         * watches historical candlestick data containing the open, high, low, and close price, and the volume of a market
+         * @param {string} symbol unified $symbol of the market to fetch OHLCV data for
+         * @param {string} timeframe the length of time each candle represents
+         * @param {int|undefined} since timestamp in ms of the earliest candle to fetch
+         * @param {int|undefined} limit the maximum amount of candles to fetch
+         * @param {array} params extra parameters specific to the hitbtc api endpoint
+         * @return {[[int]]} A list of candles ordered as timestamp, open, high, low, close, volume
+         */
+        if (limit === undefined) {
+            limit = 2;
+        }
+        const period = this.timeframes[timeframe];
```

Comment:
We should allow to pass both the unified timeframe value or the exchange accepted value
```suggestion
        const period = this.safeValue (this.timeframes, timeframe, timeframe)
```